pkg/frontend/mysql_cmd_executor.go

```diff
@@ -2386,6 +2386,15 @@ func (mce *MysqlCmdExecutor) doComQuery(requestCtx context.Context, sql string)
 		Version:      serverVersion.Load().(string),
 		TimeZone:     ses.GetTimeZone(),
 	}
+	if ses.GetTenantInfo() != nil {
```

Comment:
Why does mo_debug need to touch this?

Reply:
Because the operation on the table needs to use "AccountId"